### ISSUE TITLE
Update files to point out to "temp" folder for Windows users

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#2501]
**Environment:**
Operating System: Windows 10 Pro

**Steps to Reproduce:**
1. Follow [Development Environment setup](https://github.com/webcompat/webcompat.com/blob/master/docs/dev-env-setup.md#installing-virtualenv) guide for Windows.
2. Check logs.

**Expected Behavior:**
Logs point out to "temp" folder for Windows and "tmp" for Linux/Mac.

**Actual Behavior:**
Logs point out to "tmp" folder.

**Note:**
1. "tmp" from Linux/Mac folder in `__init__.py` file from `Config` folder (`C:\\tmp\\webcompat.log`) is "temp" folder on Windows;
2. If repository is installed on a different location (e.g D partition) a "Temp" folder needs to be manually created. 
![image](https://user-images.githubusercontent.com/12184325/41525196-d1e1e684-72e8-11e8-97f0-b2a3d8210a9a.png)
3. Update all other documents to reflect the change.

**Comment by miketaylr** [comment]
Thanks for filing @softvision-oana-arbuzov!

I'll take a stab at this.

**Comment by miketaylr** [comment]
Changing the title a bit...

**Comment by softvision-oana-arbuzov** [comment]
@miketaylr and @karlcow some little updated for `tmp` folder to be set according to the platform used.

In `__init__.py` file:
```py
def getTempPath():
    return "temp" if sys.platform is "win32" else "tmp"

LOG_FILE = os.path.join(getTempPath(), '/webcompat.log')
LOG_FMT = '%(asctime)s tracking %(message)s'
CSP_REPORTS_LOG = os.path.join(getTempPath(), '/webcompat-csp-reports.log')
```

Please review.

**Comment by miketaylr** [comment]
The code looks good, but I'm not sure it's needed. The current approach is to create a `tmp` directory in the `webcompat.com` source dir (so, `webcompat.com/tmp/`), so it doesn't really matter what platform you're on. It might be a little confusing that I used `tmp`, which looks unix-y.

Does that make sense? Or is it failing for you as-is on the latest master?

**Comment by miketaylr** [comment]
(or, it could be `webcompat.com\tmp`, on windows, but `os.path.join(os.getcwd(), 'tmp')` should take care of that. this is one of the reasons i think we should only support windows w/ linux subsystem shell -- it simplifies things a lot)

**Comment by karlcow** [comment]
yes in this case, I don't think it matters. This is just a location to save the file. Plus having two different names for `tmp` will further complicate the documentation or any scripts that would handle log rotations and stuff.

**Comment by softvision-oana-arbuzov** [comment]
@miketaylr and @karlcow that sounds good too, I will update my files and check again if it works on Windows.
